### ISSUE TITLE
License in pyproject.toml has wrong format.

### ISSUE THREAD

**Issue by flrnslbch** [Materials-Data-Science-and-Informatics/somesy#67]
**Describe the bug**
Somesy demands the license as string. At least, otherwise, I get a cryptic error message. However, when I provide a string (e.g., "MIT") this string is copied into the license field in the [project] section which is not valid in a pyproject.toml. The value of license in pyproject.toml is supposed to be a dictionary (see: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license). If license is not a dict, pip will throw an error (see below)
It would also be nice, if I could also provide the LICENSE file or the license classifier. 

**To Reproduce**
run `somesy sync` and provide "MIT" as license

**Expected behavior**
The value of license in pyproject.toml should be 
```toml
[project]
# ...
license = {text = "MIT"}
```

**Error**
When running `pip install -e .`, you get

``` 
...
        File "/tmp/pip-build-env-vp93hy2t/overlay/lib/python3.10/site-packages/pdm/backend/config.py", line 44, in __init__
          self.validate(data, root)
        File "/tmp/pip-build-env-vp93hy2t/overlay/lib/python3.10/site-packages/pdm/backend/config.py", line 65, in validate
          raise ValidationError(e.args[0], e.key) from e
      pdm.backend.exceptions.ValidationError: project.license: Field `project.license` has an invalid type, expecting a dictionary of strings (got `MIT`)
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build editable did not run successfully.
│ exit code: 1
```

**Comment by apirogov** [comment]
Thanks for your report and trying out somesy! We'll look into it right away!

**Comment by mustafasoylu** [comment]
@flrnslbch the fix is released with version 0.3.1
